ci/test.lua

```diff
@@ -408,7 +412,8 @@ local function run_test(test, status, counts)
     elseif test.private.checks ~= test.private.checks_ok then
         dfhack.printerr('test failed: ' .. test.name)
     else
-        print(('test passed in %d ms: %s'):format(elapsed_ms, test.name))
+        local elapsed_str = get_elapsed_str(elapsed_ms)
+        print(('test passed in %s ms: %s'):format(elapsed_str, test.name))
```

Comment:
@BenLubar would this cause issues with how Buildmaster parses test output? It looks like this with this change:
```
test passed in 1 ms: structures/types_meta:bitfield
test passed in <1 ms: structures/types_meta:global
test passed in <1 ms: structures/types_meta:unit
test passed in <1 ms: structures/unions:unit_action_fields
test passed in <1 ms: structures/unions:unit_action_type
test passed in <1 ms: test:internal_in_test
test passed in 1 ms: structures/find:image_set
```

(I couldn't find relevant code in https://github.com/BenLubar/build-env or https://git.lubar.me/ben/dfhack-raft)

Reply:
also the `Tests completed` line now looks like this:
```
Tests completed in 4198 ms:
```

Reply:
Let's find out!